### DISCUSSION TITLE
contact form add some fields

### DISCUSSION THREAD

**Discussion by cracksilver** [automadcms/discussions#33] [Q&A] [answered]
Hi all
Where I can add some fields like "your phone number" or "company name"
thx

**Comment by marcantondahmen** [comment, marked as answer]
Hi, that's not possible with the mail block. You will have to create a custom snippet for that and add it using the snippet block.

**Comment by cracksilver** [comment]
ok that's not really cool, I thought I could add a few fields in there somewhere. Is this really only possible with the snippets? Then I have to insert a completely new contact form using a snippet... 

But I've tried the snippets in the meantime. That's pretty cool for creating a multi-part footer ;-)
